tests/datasource/data_connector/test_configured_asset_azure_data_connector.py

```diff
@@ -228,6 +229,10 @@ def expected_batch_definitions_sorted():
     return expected
 
 
+@pytest.mark.skipif(
+    not azure_storage,
+    reason="Could not import 'storage' from azure in configured_asset_gcs_data_connector.py",
```

Comment:
accidental GCS ref here?

Reply:
Sorry about this 😅 . Silly mistake on my part.  PR has been updated